### PR TITLE
Issues/owasp dependency check

### PR DESCRIPTION
**Pull request by zcgandcomp**
Normalize naming
Add secrets for snapshot build
Add OWASP dependency check workflow

### CODE REVIEW

File: .github/workflows/maven-deploy.yml [hunk 1]
```diff
@@ -21,15 +21,13 @@ on:
...
-     - 'develop'
-     - 'test/ci'
-
-
+      - 'develop'
+      - 'test/ci'
 
 jobs:
-  maven-deploy-internal:
+  maven-deploy-push:
```
Comment (banterCZ) [inline .github/workflows/maven-deploy.yml:28]: The name `push` confuses me.

**Review by banterCZ — APPROVED** [review]

File: .github/workflows/maven-deploy.yml (hunk 1 above)
Comment (romanstrobl) [reply .github/workflows/maven-deploy.yml:28]: I agree, it is confusing because we also use `push` for the push server component in our naming.

**Review by romanstrobl — APPROVED** [review]
Approved.

File: .github/workflows/maven-deploy.yml (hunk 1 above)
Comment (zcgandcomp) [reply .github/workflows/maven-deploy.yml:28]: I will gladly rename it - this is a naming used from crypto (https://github.com/wultra/powerauth-crypto/pull/439/files)

**Pull request merged**